src/foam/dao/FixedSizeDAO.js

```diff
@@ -16,6 +16,7 @@ foam.CLASS({
     prohibitively large. Instead, this DAO will store some of that data in
     memory, but it will start to replace old data after a certain point to
     prevent infinite growth.
+    NOTE: Install via EasyDAO. It is important that when removeFromDelegate is true (the default) that this DAO is placed between the JDAO and it's delegate which is typically a MDOA or NullDAO.
```

Comment:
Why does this have to come after the JDAO? It shouldn't matter, should it? If we're recording removals in the JDAO, that's what we want anyway since when we read from the journal after a restart, the items that were removed before the restart shouldn't be accessible anyway.

Reply:
We don't really want to delete anything - consider emails or logs.   We only want to remove them from memory.   Similar with Rule History. 
Archive servers for example give us access to this older data.